packages/opencensus-instrumentation-grpc/test/test-grpc.ts

```diff
@@ -508,7 +508,7 @@ describe('GrpcPlugin() ', function() {
               grpcModule.status.OK
             );
             if (method.method === grpcClient.unaryMethod) {
-              assertStats(testExporter, 125);
+              assertStats(testExporter, 233);
```

Comment:
These changed because of the changed locked version of `object-sizeof` dependency. The actual dependency specifier is unchanged, so shouldn't effect users.